packages/code-studio/src/embed-grid/index.html

```diff
@@ -15,31 +12,17 @@
       manifest.json provides metadata used when your web app is added to the
       homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-
-    <% if (process.env.NODE_ENV === 'development') { %>
-    <link rel="icon" href="%PUBLIC_URL%/favicon-cc-app-dev.svg" />
-    <% } else { %>
-    <link rel="icon" href="%PUBLIC_URL%/favicon-cc-app.svg" />
-    <% } %>
+    <link rel="manifest" href="/manifest.json" />
```

Comment:
Hmm. I think still need some sort of public_url here no? Otherwise if embed-grid is on a path then it'll load the wrong manifest.json

Reply:
Vite automatically rewrites URLs in index.html to include the base path in the config. See [docs]('babel-plugin-add-import-extension')

The only thing it doesn't do is if you are creating strings of URLs dynamically. In those spots you need to include the env variable. I changed everywhere we had `PUBLIC_URL` (CRA variable) to `BASE_URL` (Vite variable)